phpunit.xml.dist

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
```

Comment:
We did not add any unit tests, so is it necessary to setup tests?

Reply:
At first, I added a test, but finally I deleted it. I left the base code for tests because sooner or later we will use it.